readme.md

```diff
@@ -24,8 +24,34 @@ console.log(error);
 
 console.log(serializeError(error));
 //=> {name: 'Error', message: '🦄', stack: 'Error: 🦄\n    at Object.<anonymous> …'}
+
+// Passing `maxDepth` will limit how deep we will copy the error object
+const nestedError = {...error, a: {b: 'c'}};
+console.log(serializeError(nestedError), {maxDepth: 1});
+//=> {name: 'Error', message: 'oops', stack: stack: 'Error: 🦄\n    at Object.<anonymous> …', a: {} }
 ```
 
+## API
+
+### serialize(error, options?)
+
+#### error
+
+Type: `Error`
+
+An error object.
+
+#### options
+
+Type: `object`
+
+##### maxDepth
+
+Type: `number`<br>
+Default: `Infinity`
+
+Specify a maximum depth on the serialized error object.
```

Comment:
Is it clear enough what "maximum depth" refers to? Maybe mention "properties".

I would also include some use-cases of when this option could be useful.